cmd/devp2p/internal/ethtest/eth66_suite.go

```diff
@@ -205,6 +206,35 @@ func (s *Suite) TestLargeAnnounce_66(t *utesting.T) {
 	}
 }
 
+func (s *Suite) TestOldAnnounce_66(t *utesting.T) {
```

Comment:
Isn't the code here identical to `TestOldAnnounce`, with the exception of `s.setupConnection66` instead of `s.setupConnection` ? 
I'm thinking you could maybe reduce it so there's one implementation which takes either a `setupConn` function or a boolean to distinguish which to use?